File: parser/Game.cs
public class Game
{
  public string Name { get; set; }
  public string ImageUrl { get; set; }
  public List<Genre> Genres { get; set; }

  public Game()
  {
    Name = string.Empty;
    ImageUrl = string.Empty;
    Genres = [];
  }
}

public class Genre
{
  public string Name { get; set; }

  public Genre()
  {
    Name = string.Empty;
  }
}

File: backend/entities/game.go
package entities

type Game struct {
	ID uint `gorm:"primaryKey" json:"id"` // not needed in post

	Name     string  `gorm:"size:300;index:,class:FULLTEXT,option:WITH PARSER ngram INVISIBLE" json:"name"`
	ImageUrl string  `json:"image"`
	Genres   []Genre `gorm:"many2many:game_genres;" json:"genres"`
}
